### PR TITLE
chore(android): Update globe key help and whatsnew

### PR DESCRIPTION
**Pull request by darcywong00**
This updates the Android help about the improved globe key experience from #5437 and also adds it to whatsnew.

### CODE REVIEW

File: android/help/troubleshooting/keyboard-help.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: How To - Getting Keyboard Help
 When a keyboard package is installed, it usually includes help information for using the keyboard.
 
 Step 1)
-Open the keyboard list by touching the globe key.
+Open the keyboard list by long pressing and release of the globe key.
```
Comment (mcdurdin) [inline android/help/troubleshooting/keyboard-help.md:9]:
```suggestion
Open the keyboard list by long pressing and releasing the globe key.
```

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.90-alpha](https://keyman.com/downloads/releases/alpha/15.0.90)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.90-alpha](https://keyman.com/downloads/releases/alpha/15.0.90)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.90-alpha](https://keyman.com/downloads/releases/alpha/15.0.90)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.90-alpha](https://keyman.com/downloads/releases/alpha/15.0.90)

**Pull request merged**